scripts/opentitan/autotop.py

```diff
@@ -123,40 +108,41 @@ def device_name(cls, name: str, discard_aon: bool = False) -> str:
             pname = pname.replace('_AON', '')
         return cls.DEVICE_MAP.get(pname, pname)
 
-    # pylint: disable=no-self-argument
     @classproperty
-    def languages(cls) -> list[str]:
-        """Report which language generations are supported.
+    def outkinds(cls) -> list[str]:
+        """Report which generation output formats are supported.
         """
+        # pylint: disable=no-self-argument
         prefix = 'generate_'
-        return list(sorted(f[len(prefix):]
-                    for f in dir(cls) if f.startswith(prefix)))
+        return list(sorted((f[len(prefix):]
+                           for f in dir(cls) if f.startswith(prefix)),
+                    key=lambda k: '' if k == 'qemu' else k))
```

Comment:
Maybe use `.removeprefix()` and break this up a bit?

```suggestion
        kinds = (f.remove_prefix(prefix) for f in dir(cls) if f.startswith(prefix))
        return list(sorted(kinds, key=lambda k: '' if k == 'qemu' else k'))
```

Reply:
You'right, done